### PR TITLE
create recylerviews

### PR DESCRIPTION
**Pull request by anhmv-1205**
## Create recylerviews

https://edu-redmine.sun-asterisk.vn/issues/13007

#### Screenshots
image

### CODE REVIEW

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/res/layout/item_movie.xml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            android:layout_marginBottom="@dimen/dp_16"
+            android:layout_marginStart="@dimen/dp_8"
+            android:text="@{String.valueOf(movie.getCustomVoteAverage)}"
+            android:textColor="@android:color/white"
+            android:textSize="@dimen/sp_14"
+            app:layout_constraintBottom_toBottomOf="@+id/imgPoster"
+            app:layout_constraintStart_toStartOf="@+id/imgPoster"/>
+
+        <ImageView
+            android:id="@+id/imageView2"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_movie.xml]: check convention

File: app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeViewModel.kt [hunk 2]
```diff
@@ -13,18 +14,46 @@ class HomeViewModel constructor(private val movieRepository: MovieRepository) :
...
     private fun initData() {
-        mCompositeDisposable.add(
-            movieRepository.getListMovieTopRated(Constant.PAGE_DEFAULT)
-                .subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
-                .take(1)
-                .subscribe({ movie -> movieObservable.set(movie[0]) },
-                    { throwable -> throwable.localizedMessage }))
+        mCompositeDisposable.add(movieRepository
+            .getListMovieTopRated(Constant.PAGE_DEFAULT)
+            .subscribeOn(Schedulers.io()).observeOn(AndroidSchedulers.mainThread())
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeViewModel.kt]:
```suggestion
            .subscribeOn(Schedulers.io())
            .observeOn(AndroidSchedulers.mainThread())
```

File: app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeViewModel.kt [hunk 3]
```diff
@@ -13,18 +14,46 @@ class HomeViewModel constructor(private val movieRepository: MovieRepository) :
...
+            .take(1)
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeViewModel.kt]: cái này để làm gì e

File: app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeViewModel.kt [hunk 4]
```diff
@@ -13,18 +14,46 @@ class HomeViewModel constructor(private val movieRepository: MovieRepository) :
...
+            .subscribe({ movie -> movieObservable.set(movie[0]) },
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeViewModel.kt]: reformat code

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeFragment.kt [hunk 5]
```diff
@@ -34,9 +37,16 @@ class HomeFragment : BaseFragment() {
         (activity as AppCompatActivity).setSupportActionBar(binding.toolbar)
         binding.toolbar.setTitleTextColor(Color.WHITE)
         viewModel.onStart()
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/moviedb_44/screen/home/HomeFragment.kt]: viewModel start when fragment doing onStart()

File: app/src/main/res/layout/item_movie.xml [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+        android:layout_height="wrap_content">
+
+        <ImageView
+            android:id="@+id/imgPoster"
+            android:layout_width="@dimen/dp_120"
+            android:layout_height="0dp"
+            android:layout_marginBottom="@dimen/dp_8"
+            android:layout_marginEnd="@dimen/dp_8"
+            android:layout_marginStart="@dimen/dp_8"
+            android:layout_marginTop="@dimen/dp_8"
```
Comment (tuanlvt-1117) [inline app/src/main/res/layout/item_movie.xml]:
```suggestion
            android:layout_margin="@dimen/dp_8"
```

**Comment by anhmv-1205** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**